cmd/kueuectl/app/passthrough/passthrough.go

```diff
@@ -27,53 +27,76 @@ import (
 
 type passThroughType struct {
 	name    string
+	short   string
```

Comment:
https://github.com/kubernetes-sigs/kueue/pull/2992#discussion_r1765076221

Reply:
>  and define a different one for command if necessary (but for mentioning that it's a pass to kubectl is sufficient)/

Sorry, I missed it

But why we can't use the same struct? This is the same command but with second level. No? Or I missing something?

Reply:
Done. Thanks!